src/app/package-json.js

```diff
@@ -107,7 +108,10 @@ const packageJSON = (current, context) => {
       ]),
 
       'build:js:browserify': scriptify([
-        'browserify . -s %s -o dist/%s.js'
+        'browserify .',
+        '-t babelify',
```

Comment:
browserify should depend on running the babel compiler instead of specifying the transform here.